Makefile

```diff
@@ -0,0 +1,25 @@
+SERVICE_NAME=deployments
+GO=go
+GOLIST=$(GO) list ./...
+GOBUILD=$(GO) build
+GOCLEAN=$(GO) clean
+GOTEST=$(GO) test
+
+all: test build
```

Comment:
I saw that @tranchitella had already done it like this in the workflows repository, but I would argue that it's more common that the first target (being the default if you only specify `make`) only builds. `all` could still do everything though, and the first target can be `default: build` or something like that. No strong feelings, but I thought I'd mention it, since it is how it is done in the client repositories and many "traditional" automake-based repositories and such.

Reply:
@kacf sounds reasonable; the main purpose of my Makefile in workflows was simplifying my development workflow, thus I wanted it to test and, if it is successful, to build.